packages/headless/src/api/knowledge/stream-answer-api.ts

```diff
@@ -239,49 +232,46 @@ export const answerApi = answerSlice.injectEndpoints({
               'Accept-Encoding': '*',
             },
             fetch,
-            onopen: onOpenStream,
             onmessage: (event) => {
               updateCachedData((draft) => {
                 updateCacheWithEvent(event, draft);
               });
             },
-            onerror: onError,
+            onerror: (error) => {
+              throw error;
+            },
```

Comment:
I the rare occurence were the answerAPI itself will return an error. This will likely never be used by the stream itself.